### PR TITLE
Scripting assigment

### PR DESCRIPTION
**Pull request by l0051**

### CODE REVIEW

File: scripts/countlines.sh [hunk 1]
```diff
@@ -2,32 +2,34 @@
...
 if [ ! -e $1 ] && [ ! -e ./$1 ]
 then
-	echo An argument is not a path
+	echo $error_arg_is_not_path
 	exit
 fi
 
-warning='An argument must be a regular file not directory'
-
 if [ -e ./$1 ]
```
Comment (KarenS88) [inline scripts/countlines.sh]: maybe this check is unnecessary

File: scripts/countlines.sh [hunk 2]
```diff
@@ -2,32 +2,34 @@
...
 then
 	if [ -d ./$1 ]
 	then
-		echo $warning
+		echo $error_arg_is_directory
 	else
 		wc -l < ./$1
 	fi
 else
```
Comment (KarenS88) [inline scripts/countlines.sh:24]: this case is also redundant

File: scripts/countlines.sh [hunk 3]
```diff
@@ -2,32 +2,34 @@
 
 # Count lines in file given in argument
 
+error_arg_is_directory='An argument must be a regular file not directory'
+error_one_arg='Must be one argument'
+error_arg_is_not_path='An argument is not a path'
```
Comment (KarenS88) [inline scripts/countlines.sh]: change message 'An argument is not a file path'

File: scripts/countlines.sh [hunk 4]
```diff
@@ -2,32 +2,34 @@
...
+
 if [ $# != 1 ]
 then
-	echo Must be one argument 
+	echo $error_one_arg
```
Comment (KarenS88) [inline scripts/countlines.sh]: use double quotes for echo messages, for example echo "$error_one_arg"

File: scripts/countlines.sh [hunk 5]
```diff
@@ -2,32 +2,34 @@
...
 then
-	echo Must be one argument 
+	echo $error_one_arg 
 	exit
 fi
 
 if [ ! -e $1 ] && [ ! -e ./$1 ]
 then
-	echo An argument is not a path
+	echo $error_arg_is_not_path
```
Comment (KarenS88) [inline scripts/countlines.sh]: use double quotes for echo messages

File: scripts/countlines.sh [hunk 6]
```diff
@@ -2,32 +2,34 @@
...
 fi
 
-warning='An argument must be a regular file not directory'
-
 if [ -e ./$1 ]
 then
 	if [ -d ./$1 ]
 	then
-		echo $warning
+		echo $error_arg_is_directory
```
Comment (KarenS88) [inline scripts/countlines.sh]: use double quotes for echo messages

File: scripts/ird.sh [hunk 7]
```diff
@@ -14,18 +14,16 @@ then
         exit
 fi
 
-if [ ! -e $1 ]
+if [ ! -e "$1" ]
 then
         echo $error_m_no_path
```
Comment (KarenS88) [inline scripts/ird.sh]: use double quotes for echo messages

File: scripts/ird.sh [hunk 8]
```diff
@@ -14,18 +14,16 @@ then
...
-if [ -e $1 ]
+if [ -d "$1" ]
 then
-        if [ -d $1 ]
-        then
-                echo $m_dir
-        else
-                echo $m_reg
-        fi
+	echo $m_dir
```
Comment (KarenS88) [inline scripts/ird.sh]: use double quotes for echo messages

File: scripts/ird.sh [hunk 9]
```diff
@@ -14,18 +14,16 @@ then
...
+else
+	echo $m_reg
```
Comment (KarenS88) [inline scripts/ird.sh]: use double quotes for echo messages

File: scripts/countlines.sh (hunk 3 above)
Comment (l0051) [reply scripts/countlines.sh]: +

File: scripts/countlines.sh [hunk 10]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Count lines in regular file given in argument
+
+error_arg_is_directory='An argument must be a regular file not directory'
+error_one_arg='Must be one argument'
+error_arg_is_not_path='An argument is not a file path'
```
Comment (KarenS88) [inline scripts/countlines.sh]: also change the variable name

**Pull request merged**